recipes-kernel/linux/5.4/patches/blktap2.patch

```diff
@@ -1130,16 +1122,10 @@ PATCHES
 +	struct blktap_device *tapdev = &tap->device;
 +	struct request_queue *q = tapdev->gd->queue;
 +
-+	blk_queue_flag_clear(QUEUE_FLAG_STOPPED, q);
 +	spin_lock_irq(&tapdev->lock);
-+
-+	do {
-+		struct request *rq = __blktap_next_queued_rq(q);
-+		if (!rq)
-+			break;
-+
-+		__blktap_end_queued_rq(rq, -EIO);
-+	} while (1);
++	// Moved inside lock like it was in 4.14
```

Comment:
@rneilturner When you did the 4.19 uprev, you moved `blk_queue_flag_clear(QUEUE_FLAG_STOPPED, q);` outside the `spin_lock_irq(&tapdev->lock);`.  It wasn't mentioned in the commit message.  You you have more information on why you made that change?